### PR TITLE
Update TESTS.md

### PR DESCRIPTION
**Pull request by uturunku1**
## Description

A member of the VCS team was trying to run some tests locally. These tests required the `GITHUB_POLICY_SET_IDENTIFIER` env var. They tried creating their own policy sets repo but they were having some difficulty setting that up, so I recommended that, for their purposes, to just point to an example (public) repo identifier that contains policy sets configuration, such as `https://github.com/hashicorp/test-policy-set`. They forked this repo and were able to run the tests locally. They asked me to add this option for setting `GITHUB_POLICY_SET_IDENTIFIER` to our docs.

## Output from tests
Including output from tests may require access to a TFE instance. Ignore this section if you have no environment to test against.

```
$ TFE_ADDRESS="https://example" TFE_TOKEN="example" go test ./... -v -run TestFunctionsAffectedByChange

...
```

### CODE REVIEW

**Review by annawinkler — COMMENTED** [review]
Thank you for adding this update! I have a small suggestion for the test so there is a clickable link for the repo.

File: docs/TESTS.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ Your policy set repository will need the following:
 1. A policy set stored in a subdirectory `policy-sets/foo`
 1. A branch other than `main` named `policies`
 
+If you are okay with using an example repository for policy sets, you may fork `https://github.com/hashicorp/test-policy-set` and set `GITHUB_POLICY_SET_IDENTIFIER` to the forked repository identifier `your-github-handle/test-policy-set`.
```
[the remark this answers is not in the document]
Comment (uturunku1) [reply docs/TESTS.md:13]: Much better, thanks!

**Review by annawinkler — DISMISSED** [review]
🌟

**Review by brandonc — COMMENTED** [review]
I'm thinking we should just use the value "https://github.com/hashicorp/test-policy-set" as the default and take away this confusion. WDYT?

**Comment by uturunku1** [comment]
> I'm thinking we should just use the value "https://github.com/hashicorp/test-policy-set" as the default and take away this confusion. WDYT?

This specific value would work for some of us, but not everyone. The user needs writing or admin permissions to the repo identifier, otherwise they will get a failure when running their tests that says something like "this identifier does not exist". So that is why is safest to recommend that they fork the repo.

**Review by brandonc — DISMISSED** [review]
Nope, that sounds good to me! I was not aware that you needed write access for that repo/config!

**Comment by brandonc** [comment]
I merged the test fixes in [another PR](https://github.com/hashicorp/go-tfe/pull/609)

**Review by brandonc — APPROVED** [review]

**Pull request merged**